src/options/Keybindings/default.json

```diff
@@ -654,6 +654,123 @@
         "clipboardNewWindow": [],
         "clipboardIncognitoWindow": []
       }
+    },
+    {
+      "name": "VimFx",
+      "options": {
+        "physicalKeys": false,
+        "ignoreModifierKeys": false,
+        "toggleHelpMenu": [[{ "code": "Slash", "key": "?", "shiftKey": true }]],
+        "toggleSaka": [],
+        "openLink": [[{ "code": "KeyF", "key": "f" }]],
+        "openLinkInBackgroundTab": [
+          [{ "code": "KeyF", "key": "F", "shiftKey": true }]
+        ],
+        "openLinkInForegroundTab": [{ "code": "KeyE", "key": "e"},{ "code": "KeyT", "key": "t"}],
+        "openLinkInNewWindow": [{ "code": "KeyE", "key": "e"},{ "code": "KeyW", "key": "w"}],
+        "openLinkInIncognitoWindow": [{ "code": "KeyE", "key": "e"},{ "code": "KeyP", "key": "p"}],
+        "downloadLink": [],
+        "focusLink": [],
+        "focusInput": [
+		  [{ "code": "KeyE", "key": "e" }, { "code": "KeyF", "key": "f" }]
+        ],
+        "hintChars": "sadfjklewcmpgh",
+        "scrollDown": [
+          [{ "code": "KeyJ", "key": "j" }]
+        ],
+        "scrollUp": [
+          [{ "code": "KeyK", "key": "k" }]
+        ],
+        "scrollLeft": [[{ "code": "KeyH", "key": "h" }]],
+        "scrollRight": [[{ "code": "KeyL", "key": "l" }]],
+        "scrollPageDown": [[{ "code": "Space", "key": "Space" }]],
+        "scrollPageUp": [[{ "code": "Space", "key": "Space", "shiftKey": true }]],
+        "scrollHalfPageDown": [[{ "code": "KeyD", "key": "d" }]],
+        "scrollHalfPageUp": [[{ "code": "KeyU", "key": "u" }]],
+        "scrollToBottom": [[{ "code": "KeyG", "key": "G", "shiftKey": true }]],
+        "scrollToTop": [
+          [{ "code": "KeyG", "key": "g" }, { "code": "KeyG", "key": "g" }]
+        ],
+        "scrollToLeft": [],
+        "scrollToRight": [],
```

Comment:
If this works the same as VimFx's `scroll_to_right` command, the default VimFx shortcut is `$`.

Reply:
great